lib/grape/dsl/configuration.rb

```diff
@@ -64,6 +64,9 @@ def desc(description, options = {}, &config_block)
             end
 
             config_class.configure(&config_block)
+            unless options.empty?
+              warn "[DEPRECATION] passing a options hash and a block to `desc` is deprecated. Move all hash options to block"
```

Comment:
This is good, although I think the message should be a full phrase, so "Passing ... " and with a period at the end of "Move ...".